### DISCUSSION TITLE
Media Search

### DISCUSSION THREAD

**Discussion by atinsley67** [tinacms/tinacms#4036] [Ideas]
Hi all,

I've had a request from my users to allow them to search for images in their (Cloudinary) library. The current interface of scrolling left and right is a little basic and once you have more than a few pages to click through, a image filename search would speed up the workflow of adding existing images to new content etc.

For example searching "dog" or "swim" would match "dog_going_for_a_swim_in_the_lake.jpg"

This seems timely, given the recent introduction of content search!

Thanks
Alex

**Comment by jipiboily** [comment]
Using repo-based media here, but really need this too.
